src/main/java/software/amazon/event/kafkaconnector/mapping/TimeMapper.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package software.amazon.event.kafkaconnector.mapping;
+
+import java.time.Instant;
+import org.apache.kafka.connect.sink.SinkRecord;
+import software.amazon.event.kafkaconnector.EventBridgeSinkConfig;
+
+public interface TimeMapper {
+  Instant getTime(SinkRecord sinkRecord);
+
+  void configure(EventBridgeSinkConfig eventBridgeSinkConfig);
```

Comment:
See above - might be removed

Reply:
Also see discussion in: https://github.com/awslabs/eventbridge-kafka-connector/issues/366#issuecomment-2291285866